rocana-release.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Curious why we need this release script. Couldn't we do the same thing in the POM file using maven release plugin?